requirements.txt

```diff
@@ -0,0 +1,10 @@
+Cython>=0.28,<1.0; python_version == '3.7'
+PyStemmer==1.3.0
+bblfsh==2.12.7
```

Comment:
Let's require bblfsh v3

Reply:
I'm assuming you use semantic versioning and asked for bblfh >= 3.0.0

Reply:
given the state of bblfsh 3 it will probably require a separate PR (probably not small)

Reply:
Fair enough

Reply:
moving back to 2.12.7 then